contracts/FundMe.sol

```diff
@@ -4,41 +4,61 @@ pragma solidity ^0.8.0;
 import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
 import "./PriceConverter.sol";
 
+error FundMe__NotOwner();
+
+/**
+ * @title A contract for crowd funding
+ * @author Patrick Collins
+ * @notice This contract is to demo a sample funding contract
+ * @dev This implements price feed as our library
+ */
 contract FundMe {
   using PriceConverter for uint256;
 
-  mapping(address => uint256) public s_addressToAmountFunded;
-  address[] public s_funders;
-  address public s_owner;
-  AggregatorV3Interface public s_priceFeed;
+  address private immutable i_owner;
+
+  uint256 public constant MINIMUM_USD = 50 * 1e18;
```

Comment:
Created publicly accessible constant `MIMIMUM_USD` for the transparency purposes.